methods/transient/Transient_approach1.ipynb

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
<p>Need some introduction about what this approach is doing, including the fact that user has to run the <code>Transient_base</code> notebook first. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>